libopae/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@
 
 project(libopae-c)
 
+
```

Comment:
Do we need this extra line? Can we remove it if not? This way this file will not be included in the list of files that have been changed in this PR.